### DISCUSSION TITLE
Calibre 5.29 + DRM 7.21 from AZW to PDF ... not successful

### DISCUSSION THREAD

**Discussion by performbetter** [apprenticeharper/DeDRM_tools#1842] [Q&A]
Hi there,
I keep getting "Convertion not possible " trying to convert an AZW to PDF.
I am using Windows 10
Reinstallation did not help.
It worked on the old PC but with the new one it does not. 
Any ideas? Thanks a lot.

Marlon

**Comment by performbetter** [comment]
And yes, I know, that the ususal answer is to delete the ebook, and reimport it. - That did not help.
Delete and reinstall Calibre - Did not help.
...

**Reply by ElleKayEm** [reply]
Not enough info given to say what the solution might be, but you might find one here:  https://www.mobileread.com/forums/showthread.php?t=283371
